llvm/tools/llvm-cas/llvm-cas.cpp

```diff
@@ -35,6 +35,11 @@ static cl::opt<bool> AllTrees("all-trees",
 static cl::list<std::string> PrefixMapPaths(
     "prefix-map",
     cl::desc("prefix map for file system ingestion, -prefix-map BEFORE=AFTER"));
+static cl::opt<bool>
+    CASIDFile("casid-file",
+              cl::desc("Input is CASID file, just ingest CASID."));
+static cl::list<std::string> InputFiles(cl::Positional,
```

Comment:
Maybe just call this `Inputs` since it can be CASID or FilePath depends on the action.